lib/sles4sap/azure_cli.pm

```diff
@@ -1374,3 +1378,163 @@ sub az_resource_delete {
 
     assert_script_run(join(' ', @az_command), timeout => $args{timeout});
 }
+
+=head2 az_storage_blob_upload
+
+    az_storage_blob_upload(
+        container_name=>'somecontainer',
+        storage_account_name=>'storageaccount',
+        file=>'somefilename' [, timeout=>42]);
+
+Uploads file to a storage container.
+
+=over 4
+
+=item B<container_name> Existing storage container name.
+
+=item B<storage_account_name> Storage account name.
+
+=item B<file> File to upload.
+
+=item B<timeout> Timeout for az command.
```

Comment:
Other functions in this file are indicating the default value in the documentation

Reply:
fixed